tests/test_wait_signal.py

```diff
@@ -140,20 +146,29 @@ def test_foo(qtbot):
                 pass
     """
     )
-    res = testdir.runpytest()
+
+    if configkey == "qt_wait_signal_raising" and configval is not None:
+        with pytest.warns(DeprecationWarning):
```

Comment:
This already should make sure the warning is shown with `qt_wait_signal_raising = false` in the ini as well, as there's a parametrization with `configval = "false"`.